lib/execute-method-map.ts

```diff
@@ -604,7 +604,21 @@ export const executeMethodMap = {
     params: {
       optional: ['opts'],
     }
-  }
+  },
+
+  'mobile: bluetooth': {
+    command: 'mobileBluetooth',
+    params: {
+      optional: ['opts'],
```

Comment:
Should this be `required: ['action']` to follow the README's required?

Reply:
currently the validation of execute method parameters in this driver does not really work (and never worked). Would you have time to fix it?